docs/changelog/1169.md

```diff
@@ -0,0 +1,6 @@
+
+- Added API methods for entering gradient data in the solver interface. Methods available are for scalar gradient data, vector gradient data, block scalar gradient data and block vector gradient data
+- Added integration tests for serial execution: to test read and write consistent mapping for scalar and vector data and test the different block methods
+- Added integration tests for parallel execution: to test read consistent mapping for scalar and vector data.
+- Changed the coupling scheme base class to additionally send gradient data 
+- Change the m2n communication to send gradient data
```

Comment:
Item 2 and 3 are currently not understandable without item 1. Typically, we should be able to sort such changelog entries arbitrarily (e.g. alphabetically). We could either add more information or create a hierarchy of information here.

```
- Added nearest-neighbor-gradient mapping.
  - Added integration tests ...
  - ...
```